Mobile Buy SDK Sample Apps/Mobile Buy SDK tvOS Sample/Mobile Buy SDK tvOS Sample/ViewController.swift

```diff
@@ -46,6 +46,29 @@ class ViewController: UIViewController, DataProviderSetter {
 extension ViewController: UICollectionViewDelegate {
     
     func collectionView(_ collectionView: UICollectionView, canFocusItemAt indexPath: IndexPath) -> Bool {
+        if collectionView != self.collectionView {
+            return true
+        }
         return false
     }
+    
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        if collectionView != self.collectionView {
+            let cell = collectionView.cellForItem(at: indexPath) as! ProductCollectionViewCell
+            if cell.productItem.productImages.count > 0 {
+                let pageViewController = PageViewController(product: cell.productItem)
+                self.present(pageViewController, animated: true, completion: nil)
+            } else {
+                let title = "Oops!"
```

Comment:
Suggestion: extract the alert controller creation/prep to its own method, but keep the `present()` call here.